content/setup-ubuntu-linux-for-laravel-developer.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Setup Ubuntu Linux for Laravel & Go Developer
+desc: Linux is 🤝, so I made this one with 🧑🏾‍💻.
```

Comment:
description is not rendered. Manjaro KED default font setup di chrome.
![image](https://user-images.githubusercontent.com/44990885/152489751-9eefd311-5235-4143-af91-3639c6c73569.png)

Reply:
![image](https://user-images.githubusercontent.com/48281037/152491797-205d0c49-8358-4c53-bad5-b7a7a19ef7a4.png)


Ubuntu render sih :D

Reply:
install noto-fonts-emoji

Reply:
Noto series oke